core/src/main/java/tc/oc/pgm/flag/FlagParser.java

```diff
@@ -50,29 +56,81 @@ private void checkDeprecatedFilter(Element el) throws InvalidXMLException {
     }
   }
 
-  public Post parsePost(Element el) throws InvalidXMLException {
+  private void checkDeprecatedMultiPost(Element el) throws InvalidXMLException {
+    if (el.getChildren("post").size() <= 1) return;
+    throw new InvalidXMLException(
```

Comment:
Why can we not support this anymore?

Reply:
The docs have always stated that the flag should have *one* post element (or attribute), it was a mistake to allow multiple posts to be defined within a flag (in #524), and it forces things like `sequential=true"` to be defined in the flag, which is far from ideal. The flag is not sequential, the *post* is what cycles sequentially.

Old maps will still work, it affects maps using multiple posts created within the timeframe between 524 and this pr, and @calcastor who is heavily involved in map creation has already updated the syntax for the maps affected (see comments in this pr).